pkg/apis/pipeline/v1/pipelinerun_types.go

```diff
@@ -563,3 +563,13 @@ func (pr *PipelineRun) GetTaskRunSpec(pipelineTaskName string) PipelineTaskRunSp
 	}
 	return s
 }
+
+// PipelineTaskRunTemplate is used to specify run specifications for all Task in pipelinerun.
+type PipelineTaskRunTemplate struct {
+	// +optional
+	Metadata *PipelineTaskMetadata `json:"metadata,omitempty"`
```

Comment:
let's not add metadata yet, since we don't have a way of implementing it until we swap the stored version over to v1. does that make sense?

Reply:
Yes I see. Thanks!